package.json

```
@@ -4,19 +4,23 @@
   "private": true,
   "dependencies": {
     "axios": "1.8.1",
+    "bootstrap-icons": "^1.11.3",
```

```suggestion
    "bootstrap-icons": "1.11.3",
```

Versiya problemi yarada bilər. (Fatimə Kərimova dedi)